<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAudioOptions</class>
 <widget class="QDialog" name="CAudioOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/icons/audio-card.svg</normaloff>:/icons/icons/audio-card.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab0">
      <attribute name="title">
       <string>FFT</string>
      </attribute>
      <attribute name="toolTip">
       <string>Audio FFT settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="fftSplitLabel1">
           <property name="toolTip">
            <string>Adjust split between pandapter and waterfall</string>
           </property>
           <property name="text">
            <string>Pandapter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="fftSplitSlider">
           <property name="toolTip">
            <string>Adjust split between pandapter and waterfall</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fftSplitLabel2">
           <property name="toolTip">
            <string>Adjust split between pandapter and waterfall</string>
           </property>
           <property name="text">
            <string>Waterfall</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="pandRangeLabel">
           <property name="toolTip">
            <string>Adjust dB range on the pandapter</string>
           </property>
           <property name="text">
            <string>Pandapter dB range</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkRangeSlider" name="pandRangeSlider">
           <property name="toolTip">
            <string>Adjust dB range on the pandapter</string>
           </property>
           <property name="minimum">
            <number>-120</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>-80</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="wfRangeLabel">
           <property name="toolTip">
            <string>Adjust dB range on the waterfall</string>
           </property>
           <property name="text">
            <string>Waterfall dB range</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkRangeSlider" name="wfRangeSlider">
           <property name="toolTip">
            <string>Adjust dB range on the waterfall</string>
           </property>
           <property name="minimum">
            <number>-120</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>-80</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="audioLockButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Lock panadapter and waterfall sliders together</string>
           </property>
           <property name="statusTip">
            <string>Lock panadapter and waterfall sliders together</string>
           </property>
           <property name="text">
            <string>Lock</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Recording</string>
      </attribute>
      <attribute name="toolTip">
       <string>Recording settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="recDirLabel">
         <property name="text">
          <string>Location for recorded audio files:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="recDirEdit">
           <property name="toolTip">
            <string>Enter a location for recordings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="recDirButton">
           <property name="toolTip">
            <string>Select a location using a browser</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <attribute name="toolTip">
       <string>Network streaming settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="portLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Network port to stream to</string>
           </property>
           <property name="text">
            <string>UDP port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="udpPort">
           <property name="toolTip">
            <string>Network port to stream to</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>7355</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="udpHost">
           <property name="toolTip">
            <string>Network host to stream to</string>
           </property>
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="hostLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Network host to stream to</string>
           </property>
           <property name="text">
            <string>UDP host</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="udpStereo">
           <property name="text">
            <string>Stereo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>IQ Stream</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkRangeSlider</class>
   <extends>QSlider</extends>
   <header>qtgui/ctk/ctkRangeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
